src/app/ui/universal-dapp-ui.js

```diff
@@ -176,13 +180,13 @@ UniversalDAppUI.prototype.renderInstanceFromABI = function (contractABI, address
     let calldata = calldataInput.value
     if (calldata) {
       if (calldata.length < 2 || calldata.length < 4 && helper.is0XPrefixed(calldata)) {
-        return setLLIError('the calldata should be a valid hexadecimal value with size of at least one byte.')
+        return setLLIError('The calldata should be a valid hexadecimal value with size of at least one byte.')
```

Comment:
`with size of at least one byte.` should be removed to

Reply:
This message is fine here according to `calldata.length` check

Reply:
ok